src/components/Home/disabledButton.tsx

```diff
@@ -0,0 +1,13 @@
+type Props = {
+  buttonText: string
+}
+
+function DisabledButton(props: Props) {
+  return (
+    <button className='relative m-2 h-[30px] w-[120px] cursor-default rounded-lg bg-dark-gray text-lg font-semibold text-white'>
```

Comment:
This function is used to create a grey button when a user has not filled in all the information in the form (its like a button with no functionality), so we called it the DisabledButton because the user cannot use it unless they have filled in the information: 

![image](https://user-images.githubusercontent.com/104259880/180644670-38be93bf-63fe-41d4-a844-d13c7c3ebbf3.png)

In the image, the start visit cannot be pressed (its disabled)

Reply:
O, I see. You can also use the `disabled` prop in the button component.
For example:
```tsx
{!visitStarted && !confirmation && (
          <button
            className='relative m-2 h-[30px] w-[120px] rounded-lg bg-dark-red text-lg font-semibold text-white
                                 disabled:bg-dark-gray disabled:cursor-default'
            onClick={() => {
              setVisit(true), setRunning(true)
              //setStartTime(Timestamp.now)
            }}
            disabled={!halfFilled()}
          >
            START VISIT
          </button>
        )}
```

Reply:
changed : )